packages/features/pbac/types/permission-registry.ts

```diff
@@ -0,0 +1,115 @@
+export type Resource =
+  | "booking"
+  | "eventType"
+  | "team"
+  | "organization"
+  | "insights"
+  | "availability"
+  | "workflow"
+  | "routingForm"
+  | "*";
+export type Action =
+  | "create"
+  | "read"
+  | "update"
+  | "delete"
+  | "manage"
+  | "invite"
+  | "remove"
+  | "billing"
+  | "override"
+  | "readRecordings"
+  | "*";
```

Comment:
🤔 

Might make more sense to scope these to each resource/action as the ability to have insights.readRecrding makes no sense

Reply:
Yeah this is was i initially thought of - but what happens if we want more granular permissions - can impersonate etc? 

Thats why i was thinking having CRUD + custom claims per resource

Reply:
I am thinking how do we identify which table belongs to which resource, should we have some mapping for this somewhere, to make it clear for developers as to what tables someone has write/update access to when event.update/event.create permission is given